docs/usage-manual.template.md

```diff
@@ -306,6 +309,7 @@ You can customize the browser entry code by running `$reframe eject browser`.
 We encourage you to do so and you should if you want to:
   - Initialize user tracking such as Google Analytics
   - Initialize error tracking such as Sentry
+  - Link to a frontend library such as Semantic UI
```

Comment:
Maybe "Add a frontend library such as Bootstrap or Semantic UI". While technically it's linking the user's intention is to add the lib (took me a while to understand that it's more important to describe the user's intention in his head rather than describe what's going on technically)

Reply:
yep you're right. Semantic UI they use that wording themselves

> Just link to these files in your HTML along with the latest jQuery.

https://semantic-ui.com/introduction/getting-started.html

So I thought let's do like them, but I don't mind, I'll change it :)